src/turnkeyml/analyze/util.py

```diff
@@ -100,6 +100,129 @@ def get_onnx_ops_list(onnx_model) -> Dict:
     return onnx_ops_counter
 
 
+def attribute_to_dict(attr):
+    """
+    Helper function that returns a dictionary containing node attributes
+    """
+    attr_dict = {}
+    for field in ["f", "i", "s"]:
+        if attr.HasField(field):
+            attr_dict[attr.name] = getattr(attr, field)
+            return attr_dict
+    if attr.ints:
+        attr_dict[attr.name] = list(attr.ints)
+    elif attr.floats:
+        attr_dict[attr.name] = list(attr.floats)
+    elif attr.strings:
+        attr_dict[attr.name] = list(attr.strings)
+    else:
+        attr_dict[attr.name] = "unknown_type"
+    return attr_dict
+
+
+def get_onnx_flops(onnx_model) -> Union[int, None]:
+    """
+    Calculate FLOPs found in the onnx model
```

Comment:
Would you kindly add the caveats from your PR description here?

Reply:
added in the latest commit 👍